fix: update section headings to align with v1 documentation (#1343)
auth0/docs-v2: main/docs/api/authentication/multi-factor-authentication/verify-with-out-of-band.mdx

  A code used to bind the side channel with the main channel you are using to authenticate.
</ParamField>

## Response Schema

<ResponseSchema>
<ResponseField name="access_token" type="string">
  The access token returned upon successful verification.
</ResponseField>

<ResponseField name="token_type" type="string">
  The type of token issued.
</ResponseField>

<ResponseField name="expires_in" type="integer">

>   The lifetime in seconds of the access token.

Reviewer: ```suggestion
  The access token lifetime in seconds.
```

</ResponseField>

<ResponseField name="error" type="string">

>   Error code returned when verification is pending or has failed.

Reviewer: ```suggestion
The error code for a pending or failed verification.
```
